test/connectivity/testHelper.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+function clean_up() {
+    print_highlighted_message 'Clean up'
+
+    echo 'Stop IoT Edge services'
+    systemctl stop iotedge.socket iotedge.mgmt.socket || true
+    systemctl kill iotedge || true
+    systemctl stop iotedge || true
```

Comment:
I'd rather have this function terminate a script if it fails to stop iotedge, I'm afraid it may run into a weird state when purge happen and iotedge is not completely stop. ( I'd excepted the script to fail then but idk ).

Reply:
this is more complicated; what if iotedge is not installed, calling "systemctl stop iotedge" will fail.  Current version just to do a simple thing and currently we didn't see any weird problem when calling purge.